openshift/catalogd/manifests/20-role-openshift-catalogd-metrics.yml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: catalogd-metrics-role
+  namespace: openshift-catalogd
+  labels:
+    app.kubernetes.io/name: catalogd
+    app.kubernetes.io/part-of: olm
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - endpoints
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
We may also need to provide RBAC for the `/metrics` endpoints as a non resource URL.

Reply:
IIRC that's already being provided by `prometheus-k8s` clusterrole